tests/behat/theme_boost_union_looksettings_advancedsettings.feature

```diff
@@ -0,0 +1,40 @@
+@theme @theme_boost_union @theme_boost_union_looksettings @theme_boost_union_looksettings_advancedsettings
+Feature: Configuring the theme_boost_union plugin for the "Branding" tab on the "Look" page
+  In order to use the features
+  As admin
+  I need to be able to configure the theme Boost Union plugin
+
+  Background:
+    Given the following "users" exist:
+      | username |
+      | student1 |
+      | teacher1 |
+    And the following "courses" exist:
+      | fullname | shortname |
+      | Course 1 | C1        |
+    And the following "course enrolments" exist:
+      | user     | course | role           |
+      | teacher1 | C1     | editingteacher |
+      | student1 | C1     | student        |
+    And the following "activity" exists:
+      | activity | h5pactivity          |
+      | name     | H5P package          |
+      | intro    | Test H5P description |
+      | course   | C1                   |
+      | idnumber | h5ppackage           |
+
+  @javascript @_file_upload
```

Comment:
This test does not really help in my point of view - it just sets the setting, but can't check if it has any effect. I will check if I can go beyond what you have begun here.